packages/validator/src/validators/regexpValidator.ts

```diff
@@ -2,19 +2,19 @@ import * as logger from '../utils/logger';
 import { Log } from '../types';
 
 /**
- * Checks if key or value matches regular expression in specsStruct
+ * Checks if key or value matches regular expression in template
  * @param specs - specification that is being validated
- * @param specsStruct - validator specification structure containing the regular expression
+ * @param template - template containing the regular expression
  * @param paramPath - string of parameters separated by ".", representing path to current specs location
  * @param isKeyRegexp - determines if key or value should be checked
  * @returns incorrect formatting message if the parameter key or value did not match regular expression
  */
-export function validateRegexp(specs: any, specsStruct: any, paramPath: string, isKeyRegexp = false): Log[] {
+export function validateRegexp(specs: any, template: any, paramPath: string, isKeyRegexp = false): Log[] {
   const messages: Log[] = [];
 
   if (isKeyRegexp) {
     for (const item of Object.keys(specs)) {
-      if (!item.match(new RegExp(specsStruct['__keyRegexp']))) {
+      if (!item.match(new RegExp(template['__keyRegexp']))) {
         messages.push(
           logger.error(`Key ${item} in ${paramPath}${paramPath ? '.' : ''}${item} is formatted incorrectly`)
```

Comment:
nit: I'd change the error message to "... doesn't match regex" instead of "is formatted incorrectly."

Reply:
If I understood correctly how validator will be used, the end user who is checking his specification json, doesn't have to necessarily know anything about some regex or even a validator template. So I don't think mentioning regex is a good idea, but I certainly agree in many cases the messages are way too general and have no real informative value for the user. I already opened #215, in which the idea is to implement the ability of telling user what exactly is wrong and to what it should be changed.

Reply:
Ahh, I see. Thanks.